rpm/systemd/diamond.service

```diff
@@ -2,8 +2,9 @@
 Description=diamond - A system statistics collector for graphite
 
 [Service]
-ExecStart=/usr/bin/python /usr/bin/diamond --log-stdout --foreground
-Restart=on-abort
+ExecStart=/usr/bin/python /usr/bin/diamond -p /var/run/diamond.pid
```

Comment:
@davidballano keep in mind that this will override the `pid_file` setting in the conf file if given